sos/report/plugins/krb5.py

```diff
@@ -33,10 +35,32 @@ def setup(self):
             f"{self.kdcdir}/kdc.conf",
             "/var/log/kadmind.log"
         ])
+        self.collect_kinit()
         self.add_copy_spec("/var/log/krb5kdc.log", tags="kerberos_kdc_log")
         self.add_cmd_output(f"klist -ket {self.kdcdir}/.k5*")
         self.add_cmd_output("klist -ket /etc/krb5.keytab")
 
+    def collect_kinit(self):
+        """
+        Collect the kinit command output for the system with id_provider "AD"
+        or "IPA" domains.
+        """
+
+        hostname = socket.gethostname()[:11].upper()
+        hostname_fqdn = socket.getfqdn()
+        sssd_conf = "/etc/sssd/sssd.conf"
+        if os.path.isfile(sssd_conf):
```

Comment:
`Plugin.path_isfile()` is a wrapper for this which accounts for any change in our sysroot, please use that.

Reply:
@Be-Rahul please update this to read the following as per @TurboTurtle's comment

```python
        if self.path_isfile(sssd_conf):
```